.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release from tags
+# Create a GitHub release when a tag is pushed.
+
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - 'anaconda-*'
+
+permissions:
+  contents: write
+
+jobs:
+  release-from-tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract version from the tag
+        id: get_version
+        # If the version does not fit the regexp, we'll just fail. TODO maybe abort instead?
+        run: |
+          echo "Checking pushed tag ref ${{ github.ref }}"
+          VER=$(echo ${{ github.ref }} | perl -ne '/^refs\/tags\/anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
+          if [ -z "$VER" ] ; then
+            echo "Tag ref ${{ github.ref }} is not a valid release tag."
+            exit 1
+          else
+            echo "Tag ref ${{ github.ref }} detected as release version $VER."
+            echo "::set-output name=version::$VER"
+          fi
+
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      #- name: Build anaconda-rpm container (for RPM build)
```

Comment:
I concur. Personally I think the build is rare enough that we don't have to spare any resources. Eliminating failures due to external data out od sync sounds good too.

However we should discuss this with the whole team, obviously it's commented out for some reason...

Reply:
Honestly, I don't mind. Feel free to build it because both variants has their benefits and drawbacks.

Reply:
We rebuild it on every PR push. This would make no difference. It makes the same sense here as there.

In retrospect, it seems obvious that the container "reuse" part can't happen for us. In most places that we need the containers, we can change everything including spec file, which invalidates the premise that the environment we want will be the same.